### PR TITLE
BEMINFO-561 — bem-info: Left menu separators have been fallen

### PR DESCRIPTION
**Pull request by gela-d**

### CODE REVIEW

File: src/blocks/common.blocks/menu-list/_type/menu-list_type_default.bemtree [hunk 1]
```diff
@@ -56,21 +56,26 @@ block('menu-list').mod('type', 'default')(
...
-            };
+            if (ctx.type === 'delimeter') {
+                result = { elem: 'delimeter'};
+            }
+
+            return result;
         })
     )
-);
+);
```
Comment (tavriaforever) [inline src/blocks/common.blocks/menu-list/_type/menu-list_type_default.bemtree]: отсуп

File: src/blocks/common.blocks/menu-list/_type/menu-list_type_default.bemtree [hunk 2]
```diff
@@ -56,21 +56,26 @@ block('menu-list').mod('type', 'default')(
         ),
 
         match(function() { return this.ctx._selectVal || !this.ctx.items; })(function(){
-            var ctx = this.ctx;
+            var ctx = this.ctx,
+                result = {
```
Comment (tadatuta) [inline src/blocks/common.blocks/menu-list/_type/menu-list_type_default.bemtree]:
``` js
return ctx.type === 'delimeter' ? { elem: 'delimeter'} : {
    // ...
};
```

**Comment by gela-d** [comment]
@tadatuta @tavriaforever done

**Pull request merged**